src/MIDebugEngine/AD7.Impl/AD7Property.cs

```diff
@@ -444,7 +467,7 @@ public int GetDataBreakpointInfo160(out string pbstrAddress, out uint pSize, out
         {
             try
             {
-                pbstrAddress = _variableInformation.Address() + "," + _variableInformation.FullName();
```

Comment:
> + "," + _variableInformation.FullName(); [](http://example.com/codeflow?start=61&length=41)

I don't *think* you want this. I think [this code](https://github.com/microsoft/MIEngine/blob/00a3aee314701adb598846acd23808f1248a70ba/src/MIDebugEngine/AD7.Impl/AD7PendingBreakpoint.cs#L248) depends on the address containing this.

Reply:
Ahhhh got it, I missed the `this.AddressId` taking the FullName as well